.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### New Issue Checklist
+
+- [ ] I have tried with the latest version of OHHTTPStubs
+- [ ] I have read the [README](https://github.com/AliSoftware/OHHTTPStubs/blob/master/README.md)
+- [ ] I have read the [Using the right Swift Version of `OHHTTPStubs` for your project](https://github.com/AliSoftware/OHHTTPStubs#using-the-right-swift-version-of-ohhttpstubs-for-your-project) section
+- [ ] I have searched in the [existing issues](https://github.com/AliSoftware/OHHTTPStubs/issues?utf8=✓&q=is%3Aissue)
```

Comment:
+ [ ] I have read [the OHHTTPStubs wiki](https://github.com/AliSoftware/OHHTTPStubs/wiki) to see if there wasn't a detailed page talking about my issue